rdt/utils.py

```diff
@@ -53,7 +53,6 @@ def get_col_info(table_name, col_name, meta_file):
         if table['name'] == table_name:
             prefix = op.dirname(meta_file)
             relative_path = op.join(prefix, meta['path'], table['path'])
-            desired_table = table
             data_table = pd.read_csv(relative_path)
             break
     for field in table['fields']:
```

Comment:
Can we add an empty line here? Actually, here and in every place where indentation is reduced.

Reply:
Sure. Changes already commited